package.json

```diff
@@ -59,6 +59,7 @@
     "animation"
   ],
   "dependencies": {
+    "@types/jsdom": "^21.1.1",
```

Comment:
I think this should be a devDependency so frontend users don't end up downloading it.